api/README.md

```diff
@@ -542,6 +542,47 @@ Returns the estimation of size and time (in milliseconds) to create the census g
 | 500 | `error getting strategy information` | 5007 | 
 | 500 | `error evaluating strategy predicate` | 5026 |
 
+### GET `/strategies/{strategyID}/holders`
+Returns the list of holders with their balances for a strategy. This endpoint only works with single token strategies like default ones.
+
+**Pagination URL params**
+
+| URL key | Description | Example |
+|:---|:---|:---|
+| `pageSize` | (optional) Defines the number of results per page. By default, `1000`. | `?pageSize=2` |
+| `nextCursor` | (optional) When is defined, it is used to get the page results, going forward. By default, `""`. | `?nextCursor=0x1234` |
+| `prevCursor` | (optional) When is defined, it is used to get the page results, going backwards. By default, `""`. | `?prevCursor=0x1234` |
+
+- 📥 response:
+
+```json
+{
+    "holders": {
+        "0x1": "1",
+        "0x2": "2",
+        "0x3": "3",
+        "0x4": "4",
+        "0x...": "1000",
+    },
+    "pagination": {
+        "nextCursor": "0x5",
```

Comment:
@lucasmenendez  I do not understand the format used for prodiving the`nextCursor` and `prevCursor` parameters. Why do they start with `0x`?

Reply:
They will be addresses, the next cursor is the first address of the next page, and the prev cursor is the first address of the previous one.

Reply:
Ok. I understand now. NEvertheless it seems as a counter-intuitive pagination approach. Is this because of the lack of some external integer index?

Reply:
exactly, the tokens are indexed by `ID_chainID_externalID`